pkg/volume/iscsi/iscsi_util.go

```diff
@@ -185,14 +192,20 @@ func (util *ISCSIUtil) DetachDisk(c iscsiDiskUnmounter, mntPath string) error {
 		refCount, err := getDevicePrefixRefCount(c.mounter, prefix)
```

Comment:
also see this https://github.com/kubernetes/kubernetes/issues/41041#issuecomment-278117872